pkg/infra/kubernetes/plugin.go

```diff
@@ -42,13 +43,27 @@ func (p Kubernetes) Translate(constructGraph *core.ConstructGraph, dag *core.Res
 	// For exec units that specify their own chart, we want to render and replace
 	for dir, khChart := range klothoCharts {
 		dirToLoad := filepath.Join(p.Config.Path, dir)
-		chart, err := p.helmHelper.LoadChart(dirToLoad)
+		chartContent, err := p.helmHelper.LoadChart(dirToLoad)
+
 		if err != nil {
-			errs.Append(err)
-			continue
+			if err.Error() == "Chart.yaml file is missing" {
+				chartContent = &chart.Chart{
+					Metadata: &chart.Metadata{
+						Name:        p.Config.AppName,
+						APIVersion:  "v2",
+						AppVersion:  "0.0.1",
+						Version:     "0.0.1",
+						KubeVersion: ">= 1.19.0-0",
+						Type:        "application",
+					},
+				}
```

Comment:
👍 i wouldnt make it the appname though because someone could direct us to have multiple charts if they have multiple exec units. I would make it a combination of the execution units in the klotho chart or if you have other ideas im open to them

Reply:
Your suggestion sounds good enough to me right now.